example-templates/placeholders/target/debug/.fingerprint/placeholders-687f48df171008a8/bin-placeholders.json

```diff
@@ -0,0 +1 @@
+{"rustc":3678934074608471201,"features":"[]","target":16874239820646045381,"profile":11736316127369858332,"path":1684066648322511884,"deps":[],"local":[{"CheckDepInfo":{"dep_info":"debug/.fingerprint/placeholders-687f48df171008a8/dep-bin-placeholders"}}],"rustflags":[],"metadata":11935605190402506889,"config":2202906307356721367,"compile_kind":0}
```

Comment:
all files under `example-templates/placeholders/target` should not be included in the PR, please remove them.